go/engine/pgp_push_private.go

```diff
@@ -145,6 +145,24 @@ func (e *PGPPushPrivate) remove(m libkb.MetaContext, fs *keybase1.SimpleFSClient
 	return err
 }
 
+func (e *PGPPushPrivate) linkExists(m libkb.MetaContext, fs *keybase1.SimpleFSClient, file string) (exists bool, err error) {
+	defer m.Trace(fmt.Sprintf("linkExists(%q)", file), func() error { return err })()
+
+	var dirent keybase1.Dirent
+	dirent, err = fs.SimpleFSStat(m.Ctx(), keybase1.SimpleFSStatArg{
+		Path:                keybase1.NewPathWithKbfsPath(file),
+		RefreshSubscription: false,
+	})
+	if err != nil {
+		m.Debug("error type %T, but assuming that the file doesn't exist", err)
```

Comment:
@strib is there a way to get a richer error type here to check if it's a notfound error versus something else? I didn't see an easy way and can just opt for this hack.

Reply:
(It's just an `*errors.errorString`)

Reply:
Nah we don't have typed errors here, sorry.  You could grep for something in the string if you want, but I think logging the error is fine.  Probably you should log the error message though, not just the type, since that's generic as you say.

Reply:
To be clear, I'm fine with this hack way.